app/pods/components/course-library/doubts/template.hbs

```diff
@@ -0,0 +1,10 @@
+<div>
+<Player::PlayerDoubtsList @doubts={{@payload}} />
+</div>
+<style>
```

Comment:
styles go in `app.scss` . 

Putting them here is wrong. It is not even valid HTML (although forgiving browsers like Chrome will render this).